USAGE.md

```diff
@@ -111,7 +111,8 @@ The `Watcher` constructor can be passed 3 different options:
 
 * `time` - The time threshold
 * `ratio` - The ratio threshold
-* `rootMargin` - The [rootMargin](https://wicg.github.io/IntersectionObserver/#dom-intersectionobserverinit-rootmargin) in object form.
+* `rootMargin` - The rootMargin in object form.
```

Comment:
Why get rid of the link

Reply:
It's 404-ing. FWIW `rootMargin` is as well. Just need to find the new relative links for these.

Reply:
maybe https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API ?